projects/ngx-datatable/src/lib/themes/bootstrap.scss

```diff
@@ -111,3 +111,41 @@ bootstrap table theme
     }
   }
 }
+/**
```

Comment:
I'm not too keen with ngx-datatable's styling hierarchy, but I see a lot of redundant styles for bootstrap, dark and material's CSS files. Is there a way to share utilities and base definitions and e.g. only define/overwrite the colors within the individual themes?

Reply:
We in SiMPL-element only using bootstrap.scss. We might need some refactoring and will need to make sure that it doesn't break any style hierarchy. We should take it as separate MR rather than in this. WDYT ?

Reply:
Looping @timowolf and @kfenner here. Doing improvements in a follow-up is fine, but I personally think we should fix the redundant code already as part of this MR. Either by moving the shared SCSS code to the component styling and solely do the color overwrites on theme level or by reducing styling completely and do it on Element level. Not sure which approach is better though.

Reply:
I have moved ghost loader specific styles into its own component so no changes in existing ngx-datatable scss.
using variables so we can just override them while using it with SiMPL element

Reply:
This is done now.